Sources/Sentry/SentryTransaction.m

```diff
@@ -112,11 +117,43 @@ - (void)setOperation:(NSString *)operation
     [self.spanContext setOperation:operation];
 }
 
+- (SentrySpan *)startChildWithOperation:(NSString *)operation
+{
+    return [self startChildWithOperation:operation description:nil];
+}
+
+- (SentrySpan *)startChildWithOperation:(NSString *)operation
+                            description:(nullable NSString *)description
+{
+    return [self startChildWithParentId:self.spanId operation:operation description:description];
+}
+
+- (SentrySpan *)startChildWithParentId:(SentrySpanId *)parentId
+                             operation:(NSString *)operation
+                           description:(nullable NSString *)description
+{
+    SentrySpan *span = [[SentrySpan alloc] initWithTransaction:self
+                                                       traceId:self.traceId
+                                                      parentId:parentId];
+    span.operation = operation;
+    span.spanDescription = description;
+    span.sampled = self.isSampled;
+    @synchronized(self.spans) {
```

Comment:
`m`: Can we please add a test to make sure this is thread-safe?

Reply:
One might argue "make sure it's thread-safe" isn't possible to achieve :P

Reply:
> One might argue "make sure it's thread-safe" isn't possible to achieve :P

Indeed. Neither for positive, nor for negative outcome.

Reply:
time has shown that this answer will come, soon or later, if it was or not :D